src/pcapi/routes/adage_iframe/bookings.py

```diff
@@ -30,12 +32,26 @@ def book_educational_offer(authenticated_email: str, body: BookEducationalOfferR
         booking = educational_api.book_educational_offer(
             redactor_email=(body.redactorEmail), uai_code=body.UAICode, stock_id=body.stockId
         )
+    except AdageException:
+        logger.info("Could not book offer: adage api call failed", extra={"redactor_email": body.redactorEmail})
+        raise ApiErrors(
+            {"global": "L'appel api pour retrouver les informations du rédacteur a échoué"},
+            status_code=500,
+        )
+    except InstitutionalProjectRedactorNotFoundException:
+        logger.info(
+            "Could not book offer: redactor not found in adage database",
+            extra={
+                "redactor_email": body.redactorEmail,
+            },
+        )
+        raise ApiErrors({"global": "Le rédacteur de projet n'a pas été trouvé côté Adage"}, status_code=404)
```

Comment:
```suggestion
        raise ApiErrors({"global": "Le rédacteur de projet n'existe pas dans Adage"}, status_code=404)
```

Par contre tu utilises parfois `rédacteur` et parfois `rédacteur de projet` dans tes retours
Il faudrait uniformiser

Reply:
Je vais mettre rédacteur de projet partout